lib/mail.ts

```diff
@@ -0,0 +1,24 @@
+import nodemailer from "nodemailer";
+
+const SOURCE_EMAIL = "";
+const SOURCE_PASSWORD = "";
```

Comment:
instead of using string as literal use env variables - 

- process.env.NEXT_SOURCE_EMAIL
- process.env.NEXT_SOURCE_PASSWORD